### ISSUE TITLE
Strange values when reading DHW temperature values

### ISSUE THREAD

**Issue by Zolli** [wrongisthenewright/ebusd-configuration-ariston-bridgenet#23]
Hi, i use the current (2.4) release of the configuration, and i saw that DHW temepratures added for both reading and writing. When i try to read these values i get strange values like: -2867.1 or 3174.5
I use this config with an Ariston Nimbus Flex S90 HP system, and other values works wonderfully, expect these two.

The two value that i try to read:
- energymgr,dhw_economy_temp
- energymgr,dhw_comfort_temp

**Comment by wrongisthenewright** [comment]
Hi, 
good find! there was an error on the parsing of the read commands, it was missing the IGN:1 needed for direct ask commands.
You can alter your ariston.csv file

```
#DHW Temp settings
r,energymgr,dhw_comfort_temp,DHW Comfort Temp,,18,2000,6126,,**s,IGN:1**,,,,,,SIN,10,°C,,,,,,,,,,,,,,,,,,,,,,
...
r,energymgr,dhw_economy_temp,DHW Economy Temp,,18,2000,6226,**,s,IGN:1**,,,,,,SIN,10,°C,,,,,,,,,,,,,,,,,,,,,,
...
r,energymgr,dhw_comfort_mode,DHW Comfort Mode,,18,2000,c028,,**s,IGN:1**,,,,,,BCD,0=off;1=timed;2=always_on
...
r,energymgr,dhw_thermal_cleansing_mode,DHW Thermal Cleansing Function,,18,2000,0b20,**,s,IGN:1**,,,,,,onoff
```

I'll fix it on the repo

**Comment by wrongisthenewright** [comment]
Closed in v2.5
